sourced/ml/cmd_entries/args.py

```diff
@@ -67,3 +67,13 @@ def add_feature_args(my_parser: argparse.ArgumentParser, required=True):
             my_parser.add_argument(
                 "--%s-%s" % (ex.NAME, opt), default=val, type=json.loads,
                 help="%s's kwarg" % ex.__name__)
+
+
+def add_bow_args(my_parser: argparse.ArgumentParser):
+    my_parser.add_argument(
+        "--bow", required=True, help="[OUT] The path to the Bag-Of-Words model.")
+    my_parser.add_argument(
+        "--batch", default=BOWWriter.DEFAULT_CHUNK_SIZE, type=int,
+        help="The maximum size of a single BOW file in bytes.")
+    my_parser.add_argument(
+        "--parquet", action="store_true", help="If it's parquet input.")
```

Comment:
Please rephrase the help message

Reply:
I don't see any relation between parquet and bow here...

Reply:
there is none really, this is only added here until https://github.com/src-d/ml/pull/231 and https://github.com/src-d/ml/pull/232 are merged for consistency (232 changes the parsers in a simpler way). anyway, changing help msg to "Use Parquet files as input."